src/main/java/org/mondemand/Client.java

```diff
@@ -381,56 +381,44 @@ public void addTransportsFromDefaultConfigFile()
 
   /**
    * adds transports from a configuration file. the format of the file is:
-   * MONDEMAND_ADDR="ip_addr_1, ip_addr_2, ..."
-   * MONDEMAND_PORT="port"
-   * (optional) MONDEMAND_TTL="ttl"
+   * MONDEMAND_ADDR="<ip>[,<ip>]?"
+   * MONDEMAND_PORT="<port>[,<port>]?"
+   * (optional) MONDEMAND_TTL="<ttl>[,<ttl>]?"
+   * (optional) MONDEMAND_SENDTO="<sendto>"
    *
    * @param configFileName - configuration file name.
    * @throws Exception if file does not exist, or if there is a problem reading
    *        the file, or either port or address is missing, or if port cannot
    *        be converted to number, or if addresses cannot be converted to valid
-   *        hosts, or if ttl cannot be converted to number in valid range,
-   *        or if a transport cannot be created for addresses/port
-   *        specified in the file.
+   *        hosts, or if ttl cannot be converted to number in valid range, or if
+   *        sendto cannot be converted to number in valid range, or if length of
+   *        port or ttl arrays does not equal one or length of addr array, or if
+   *        a transport cannot be created for addresses/port specified in the
+   *        file.
    */
   public void addTransportsFromConfigFile(String configFileName)
       throws Exception {
```

Comment:
It's probably a good idea to list the specific exceptions that this method might throw rather than declaring it "throws Exception".  The above forces callers to wrap calls in a catch(Exception), which is usually best not to do.